<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>632</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1E88E5 </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_imagenUat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Logos/Archivos/log_uat_nuevo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenFi">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Logos/Archivos/FIT_logo_vertical.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_imagenGatos">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Recursos.qrc">:/Logos/Archivos/Gatos.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Sh">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Forte&quot;;
background-color: rgb(130, 0, 2);
color: rgb(255, 255, 255);
border-radius:30px;
border:6px solid #B0BEC5;</string>
    </property>
    <property name="text">
     <string>Segundos a Horas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Hs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Forte&quot;;
background-color: rgb(130, 0, 2);
color: rgb(255, 255, 255);
border-radius:30px;
border:6px solid #B0BEC5;</string>
    </property>
    <property name="text">
     <string>Horas a Segundos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Tiempo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Horas: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ValorTiempo">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Forte&quot;;
                   color: black;
                   border-radius: 10px;
                   font-weight: bold;
                   font-style: italic;</string>
    </property>
    <property name="text">
     <string>0 Segundos</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Valor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
border-radius:25px;
border:8px solid #B0BEC5;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32774</number>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Conversor de Horas (En formato de 24horas)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Unidad2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string>Unidad 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_titulo_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-decoration: underline;
font: 20pt &quot;Forte&quot;;
color:rgb(0, 0, 0);
border-radius: 10px;
font-weight: bold;
font-style: italic;</string>
    </property>
    <property name="text">
     <string> a Segundos del d√≠a</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
